micro-frontends/src/next-ui/Components/AddAllergy/AddAllergy.spec.jsx

```diff
@@ -342,6 +360,110 @@ describe("AddAllergy", () => {
       severity: { uuid: "162301AAAAAA"},
       comment: "",
     }, "patient#1");
+  });
+
+  it("should show known allergy selector when existingAllergies is empty", () => {
+    render(
+        <IntlProvider locale="en">
+          <AddAllergy
+              onClose={onClose}
+              onSave={onSave}
+              patient={patient}
+              provider={provider}
+              severityOptions={mockSeverityData}
+              allergens={mockAllergensData}
+              reaction={mockReactionsData}
+              existingAllergies={[]}
+          />
+        </IntlProvider>
+    );
+    expect(screen.getByText("Does the patient have any known allergies?")).not.toBeNull();
```

Comment:
Any specific reason you used `not.toBeNull()` instead of `toBeInTheDocument()` ?